src/test/java/edu/cornell/kfs/sys/util/CuXMLUnitTestUtils.java

```diff
@@ -0,0 +1,46 @@
+package edu.cornell.kfs.sys.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.custommonkey.xmlunit.DetailedDiff;
+import org.custommonkey.xmlunit.Diff;
+import org.custommonkey.xmlunit.Difference;
+import org.custommonkey.xmlunit.XMLUnit;
+import org.xml.sax.SAXException;
+
+public class CuXMLUnitTestUtils {
+    
+    private static final Logger LOG = LogManager.getLogger();
+    
+    public static void compareXML(File expectedXmlFile, File actualXmlFile) throws SAXException, IOException {
+        try (FileInputStream actualFileInputStream = new FileInputStream(actualXmlFile);
+                FileInputStream expectedFIleInputStream = new FileInputStream(expectedXmlFile);
+                BufferedReader actualBufferedReader = new BufferedReader(new InputStreamReader(actualFileInputStream));
+                BufferedReader expectedBufferedReader = new BufferedReader(new InputStreamReader(expectedFIleInputStream));) {
```

Comment:
For consistency with our other code and KualiCo's code, I would suggest explicitly specifying the UTF-8 charset when creating the `InputStreamReader` objects.

Reply:
with the new XML Unit libraries, I didn't need to use InputStreamReader